<launch>

    <!--- Run map tools -->
    <include file="$(find waterplus_map_tools)/launch/wpb_home_nav_test.launch" />

    <!--- Run servers -->
    <include file="$(find wpb_home_behaviors)/launch/all_servers.launch" />

    <!-- Run SR node -->
    <node pkg="xfyun_waterplus" type="iat_node" name="xfyun_iat_node" output="screen">
        <param name="cn" type="bool" value="false"/> 
        <param name="start" type="bool" value="true"/>
    </node>

    <!-- Run TTS node -->
    <node pkg="xfyun_waterplus" type="tts_node" name="xfyun_tts_node" />

    <!-- Run sound_play -->
    <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py"/>

    <!--- Run entrance_detect -->
    <node pkg="wpb_home_apps" type="entrance_detect" name="entrance_detect"/>

    <!--- Run GPSR -->
    <node pkg="wpb_home_apps" type="gpsr" name="gpsr" output="screen"/>

</launch>
